integration/cmd_type_test.go

```diff
@@ -8,84 +8,84 @@ import (
 	"github.com/tendermint/starport/starport/pkg/cmdrunner/step"
 )
 
-func TestGenerateAnAppWithStargateWithTypeAndVerify(t *testing.T) {
+func TestGenerateAnAppWithStargateWithListAndVerify(t *testing.T) {
```

Comment:
Not related to this PR but couldn't we think about simplifying the name of the integration tests? They seem very verbose to me

Reply:
I like it to make it explicit but we can think about other options.

Reply:
I agree with @lubtd, but also I like more descriptive information, as @ilgooz mentions. My suggestion is to use the less verbose names in the method name and use subs tests to describe each test. What do you think?

```go
t.Run("testing create a new list" func(t *testing.T) {
	env.Must(env.Exec("create a list",
		step.NewSteps(step.New(
			step.Exec("starport", "s", "list", "user", "email"),
			step.Workdir(path),
		)),
	))
})
```